src/test/java/com/amplitude/AmplitudeLogTest.java

```diff
@@ -56,6 +56,14 @@ public void testLog(
     assertEquals(expectedDebugLog, outContent.toString().trim());
   }
 
+  @Test
```

Comment:
nits: maybe put this test to line42 (before the code block above) as the code block above uses the code block below as an argument source.